stable/chaoskube/README.md

```diff
@@ -41,13 +41,16 @@ $ helm install stable/chaoskube --set dryRun=false
 |---------------------------|-----------------------------------------------------|-----------------------------------|
 | `name`                    | container name                                      | chaoskube                         |
 | `image`                   | docker image                                        | quay.io/linki/chaoskube           |
-| `imageTag`                | docker image tag                                    | v0.6.1                            |
+| `imageTag`                | docker image tag                                    | v0.7.1                            |
 | `replicas`                | number of replicas to run                           | 1                                 |
 | `interval`                | interval between pod terminations                   | 10m                               |
 | `labels`                  | label selector to filter pods by                    | "" (matches everything)           |
 | `annotations`             | annotation selector to filter pods by               | "" (matches everything)           |
 | `namespaces`              | namespace selector to filter pods by                | "" (all namespaces)               |
 | `dryRun`                  | don't kill pods, only log what would have been done | true                              |
+| `timezone`                | Set timezone for running actions (Optional)         | "" (UTC)                          |
+| `excludedWeekdays`        | Set Days of the Week to avoid actions (Optional)    | "" (Don't skip any days)          |
+| `excludedTimesOfDay`      | Set Time Range to avoid actions  (Optional)         | "" (Don't skip any time)          |
```

Comment:
This hasn't made it into `v0.7.x`. Let's bump the version to `v0.8.0` in this PR and I'll try to make a release in the next days.

Reply:
Ahh, my apologies.  I saw time of day exclusion in the README for your repo and made a bad assumption.  I'll bump the version to 0.8.0 and wait on your word for the next release.